src/main/java/com/jpn/lang/controller/EmployeeController.java

```diff
@@ -58,4 +59,10 @@ public ResponseEntity<?> deleteEmployee(Long id) {
         employeeService.deleteEmployee(id);
         return ResponseEntity.ok("Delete...");
     }
+
+    @Override
+    public ResponseEntity<?> put(EmployeePutRequest putEmployee, Long id) {
```

Comment:
Can you return EmployeeResponse object from here not ResponseEntity<?>? 
same as post or get.

First convert Employee into EmployeeResponse 
then return.

we will use ResponseEntity<?>  later

Reply:
I have completed this task..